pkg/streamingpromql/testutils/utils.go

```diff
@@ -15,16 +15,14 @@ import (
 
 // Why do we do this rather than require.Equal(t, expected, actual)?
 // It's possible that floating point values are slightly different due to imprecision, but require.Equal doesn't allow us to set an allowable difference.
-func RequireEqualResults(t testing.TB, expr string, expected, actual *promql.Result, checkAnnotations bool) {
+func RequireEqualResults(t testing.TB, expr string, expected, actual *promql.Result) {
```

Comment:
I had intended to keep checkAnnotations in case we needed it for other cases, but it's easy enough to add back in.